frontend/src/routes/(authenticated)/admin/AdminProjects.svelte

```diff
@@ -59,7 +59,7 @@
   $: shownProjects = limitResults ? limit(filteredProjects, lastLoadUsedActiveFilter ? DEFAULT_PAGE_SIZE : 10) : filteredProjects;
 
   let deleteProjectModal: ConfirmDeleteModal;
-  async function softDeleteProject(project: ProjectItemWithDraftStatus): Promise<void> {
+  async function softDeleteProject(project: ProjectItem): Promise<void> {
     const result = await deleteProjectModal.open(project.name, async () => {
       const { error } = await _deleteProject(project.id);
```

Comment:
Sorry, for a late comment, but how about we move the `if (project.isDraft)` logic here and rename this method to `deleteProjectOrDraft` instead of having 2 essentially duplicate methods. E.g.
const { error } = await (project.isDraft ? _deleteDraftProject(project.id) : _deleteProject(project.id));

Reply:
That's how I would prefer to write it, as I prefer functional-style code. But I've had functional-style code rejected in previous code reviews, so I'm never quite certain how much functional style is "too much". I'll switch it to this style tomorrow.

Reply:
I'm fine with this also. I would say too far would be passing a function as a parameter which actually executes the delete.